R/pkg/R/DataFrame.R

```diff
@@ -276,6 +276,57 @@ setMethod("names<-",
             }
           })
 
+#' @rdname columns
+#' @name colnames
```

Comment:
Although it is simply wrapping another method, having example and some description helps with in-line documentation.